src/cryptoadvance/specter/server_endpoints/wallets_api.py

```diff
@@ -425,15 +427,26 @@ def addressinfo(wallet_alias):
         wallet = app.specter.wallet_manager.get_by_alias(wallet_alias)
         address = request.form.get("address", "")
         if address:
-            descriptor = wallet.get_descriptor(address=address)
```

Comment:
In the old version this descriptor was a class: `<class 'embit.descriptor.descriptor.Descriptor'>`